content/en/docs/setup/_index.md

```diff
@@ -12,84 +12,101 @@ content_template: templates/concept
-your own customized cluster on your bare metal servers.
+Various solutions are available to create a Kubernetes cluster in learning and production environments.
 
 {{% /capture %}}
 
 {{% capture body %}}
 
-## Local-machine Solutions
-
-A local-machine solution is an easy way to get started with Kubernetes. You
-can create and test Kubernetes clusters without worrying about consuming cloud
-resources and quotas.
-
-You should pick a local solution if you want to:
-
-* Try or start learning about Kubernetes
-* Develop and test clusters locally
-
-Pick a [local-machine solution](/docs/setup/pick-right-solution/#local-machine-solutions).
-
-## Hosted Solutions
-
-Hosted solutions are a convenient way to create and maintain Kubernetes clusters. They 
-manage and operate your clusters so you don’t have to.  
-
-You should pick a hosted solution if you:
-
-* Want a fully-managed solution
-* Want to focus on developing your apps or services  
-* Don’t have dedicated site reliability engineering (SRE) team but want high availability
-* Don't have resources to host and monitor your clusters 
-
-Pick a [hosted solution](/docs/setup/pick-right-solution/#hosted-solutions).
-
-## Turnkey – Cloud Solutions
-
-
-These solutions allow you to create Kubernetes clusters with only a few commands and 
-are actively developed and have active community support. They can also be hosted on 
-a range of Cloud IaaS providers, but they offer more freedom and flexibility in 
-exchange for effort. 
-
-You should pick a turnkey cloud solution if you:
-
-* Want more control over your clusters than the hosted solutions allow
-* Want to take on more operations ownership 
-
-Pick a [turnkey cloud solution](/docs/setup/pick-right-solution/#turnkey-cloud-solutions)
-
-## Turnkey – On-Premises Solutions
-
-These solutions allow you to create Kubernetes clusters on your internal, secure,
-cloud network with only a few commands.
-
-You should pick a on-prem turnkey cloud solution if you:
-
-* Want to deploy clusters on your private cloud network
-* Have a dedicated SRE team
-* Have the resources to host and monitor your clusters
-
-Pick an [on-prem turnkey cloud solution](/docs/setup/pick-right-solution/#on-premises-turnkey-cloud-solutions).
-
-## Custom Solutions
-
-Custom solutions give you the most freedom over your clusters but require the 
-most expertise. These solutions range from bare-metal to cloud providers on 
-different operating systems.
-
-Pick a [custom solution](/docs/setup/pick-right-solution/#custom-solutions).
+## Learning environment
+
+If you want to get familiar with Kubernetes, you can use the Docker-based solutions. You can use tools that are supported by the Kubernetes community or the tools in the ecosystem to setup a Kubernetes cluster on a local-machine. 
+
+{{< table caption="Local-machine solutions table that lists the tools supported by the community and the ecosystem to deploy Kubernetes." >}}
```

Comment:
why is Local-machine hyphenated?

Reply:
Yes, I missed this because it is only visible to the screen readers. Updated it as suggested.